README.md

```diff
@@ -7,105 +7,71 @@
+---
 
 # Quickstart
 
-## Setup
+If you are proficient in a front end framework, creating a standalone web app is painless.
 
-This framework requires you to install [Rust](https://www.rust-lang.org/tools/install).
+To get started, you can clone our quickstart. We explain in detail [here](https://seed-rs.org/guide/quickstart).
 
-You'll need a recent version of Rust: `rustup update`
+<details>
+<summary>seed-quickstart</summary>
 
-The wasm32-unknown-unknown target: `rustup target add wasm32-unknown-unknown`
-
-And cargo-make: `cargo install --force cargo-make`
-
-## The theoretical minimum
-
-To start, clone [the quickstart repo](https://github.com/seed-rs/seed-quickstart):
-`git clone https://github.com/seed-rs/seed-quickstart.git`,
-run `cargo make build` in a terminal to build the app, and `cargo make serve` to start a dev server
-on `127.0.0.0:8000`. If you'd like the compiler to automatically check and recompile when you 
-make changes, run `cargo make watch` instead of `cargo make build`.
-
-
-## A little deeper
-
-Alternatively, create a new lib with Cargo: `cargo new --lib appname`. Here and everywhere it appears in this guide, `appname` should be replaced with the name of your app.
-
-If not using the quickstart repo, create an Html file with a body that contains this:
-
-```html
-<section id="app"></section>
-<script type="module">
-    import init from '/pkg/package.js';
-    init('/pkg/package_bg.wasm');
-</script>
-```
-
-The first line above is an empty element with id: It's where your app will render.
-The subsequent ones load your app's wasm modules.
-
-The quickstart repo includes this file. You will eventually need to modify it to
-change the page's title, add a description, favicon, stylesheet etc.
-
-`Cargo.toml`, which is a file created by Cargo that describes your app, needs `wasm-bindgen`, `web-sys`, and `seed` as dependencies, and crate-type
-of `"cdylib"`. The version in the quickstart repo has these set up already. Example:
-
-```toml
-[package]
-name = "appname"
-version = "0.1.0"
-authors = ["Your Name <email@address.com>"]
-edition = "2018"
-
-[lib]
-crate-type = ["cdylib"]
-
-[dependencies]
-seed = "^0.6.0"
-wasm-bindgen = "^0.2.50"
-```
-
-## A short example
-
-Here's an example demonstrating structure and syntax; it can be found in working form
-in the [counter example](https://github.com/seed-rs/seed/tree/master/examples/counter).
-Descriptions of its parts are in the
-Guide section below. Its structure follows [The Elm Architecture](https://guide.elm-lang.org/architecture/).
-
-_lib.rs_:
+[Quickstart repository](https://github.com/seed-rs/seed-quickstart) | [Download Rust](https://www.rust-lang.org/tools/install)
+~~~
+rustup update
+rustup target add wasm32-unknown-unknown
+cargo install --force cargo-make
+~~~
+~~~
+git clone https://github.com/david-oconnor/seed-quickstart.git
+cd seed-quickstart
+cargo make build
+cargo make serve
+~~~
+To build automatically use `cargo make watch`.
+</details>
```

Comment:
I think it would be good to have something visual on the page so we don't have to depend on someone clicking another link to see what Seed code looks like. [React's](https://github.com/facebook/react#examples) page has a super minimal example which is nice, but I assumed Seed's builder requires an update function so this seemed simple enough. Maybe we can simplify it more?
1. Use a type for model instead of struct
2. Only implement an Increment (no decrement)
3. No div, only a button with a count

Reply:
Just to make sure:
> Is it necessary? Isn't above link to quickstart enough?
- This comment was only for instructions for the Quickstart. Because you can read them in the repo, are out-dated and we'll forgot to update them again once Seeder is ready.

---

> Maybe we can simplify it more?

Sure, but we also don't want to confuse users. Modify it as you like and it feels to you as beginner-friendly.

Reply:
Oh I see. Agreed! I decided it would be best to use a `type` instead of `struct` in the excerpt for simplicity, and then in the quickstart page on the website we can explain the transition to a `struct` and implementing `default`.